lib/std/array_list.zig

```diff
@@ -24,9 +28,18 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
     }
     return struct {
         const Self = @This();
-
-        /// Content of the ArrayList
+        /// Contents of the list. Pointers to elements in this slice are
+        /// **invalid after resizing operations** on the ArrayList, unless the
+        /// operation explicitly either: (1) states otherwise or (2) lists the
+        /// invalidated pointers.
+        ///
+        /// The allocator used determines how element pointers are
+        /// invalidated, so the behavior may vary between lists. To avoid
+        /// undefined behavior, take into account the above paragraph plus the
```

Comment:
```suggestion
        /// illegal behavior, take into account the above paragraph plus the
```

See #2402 for context

Reply:
I had no clue about the preferred terminology. Thanks